### ISSUE TITLE
⌘⌫  Delete Shortcut Doesn't Work

### ISSUE THREAD

**Issue by astroud** [glushchenko/fsnotes#596]
In Version 3.2.0 (261), ⌘⌫ should delete a note (according to the options when right clicking a note), however it doesn't work.

⌥⌘⌫  is the shortcut that works to delete a note.

**Comment by glushchenko** [comment]
When cursor focused ta edit area?

**Comment by astroud** [comment]
No cursor. When I select a note, the cursor appears in the note’s field. When I click on the note a second time, the cursor disappears. Then get my system’s alert sound when I try to delete the note with ⌘⌫. When I add Alt/Option to the command, the note deletes. Right clicking and selecting Delete works fine too.

**Comment by peavine** [comment]
I'm on FSNotes 3.2.0 (261) with Mojave.

1) If the FSNotes preference option "Focus in editor when selecting note" is not enabled:

* The command-delete keyboard shortcut deletes the text from the cursor to the beginning of the line if the cursor is in the editor.

* The command-delete keyboard shortcut deletes the selected note if the cursor is not in the editor window.

2) If the FSNotes preference option "Focus in editor when selecting note" is enabled:

* The command-delete keyboard shortcut deletes the text from the cursor to the beginning of the line if the cursor is in the editor.

* The command-delete keyboard shortcut does nothing if the cursor is not in the editor (focus in Note List).

* The option-command-delete keyboard shortcut deletes the selected note regardless of whether the cursor is in the editor or not.

I don't know if FSNotes is supposed to work this way or not but it does seem a bit inconsistent. BTW, Apple considers the "delete" key to be the backspace key.

**Comment by peavine** [comment]
I installed FSNotes 3.3 and this works well. Thanks.

BTW, the Command-Delete keyboard shortcut (aka Command-Backspace keyboard shortcut) now deletes the note highlighted in the Note List no matter where the focus is at--including the Sidebar, Search or create bar, or Editor. Just something to keep in mind.

**Comment by quinncomendant** [comment]
Ok, so I thought I was going crazy, until I discovered this functionality change: `⌘`+`Delete` now deletes notes. 😳

I normally use `⌘`+`Delete` to "delete to beginning of line" but now, I'm accidently deleting my notes every 5 minutes. I had to go remove a note from the trash already 10 times today (`⌘`+`Delete` is a strong habit). 😣

Can we add a *Undo* feature to remove a note from the trash if it is deleted accidently? Currently, if you delete a note, the *Undo* menu item is greyed-out.
